src/test/java/core/basesyntax/MyCalculatorTest.java

```diff
@@ -0,0 +1,312 @@
+package core.basesyntax;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+
+class MyCalculatorTest {
+    private static MyCalculator calculator;
+
+    @BeforeAll
+    static void init() {
+        calculator = new MyCalculator();
+    }
+
+    @Test
+    void additionTwoPositive_Ok() {
+        double a = 2.67;
+        double b = 3.445;
+        double expected = a + b;
+        char add = '+';
+        assertEquals(calculator.calculate(a, b, add), expected);
```

Comment:
```suggestion
        assertEquals(calculator.calculate(a, b, ADDITION_OPERATOR), expected);
```
краще робити код консистентним щоб всюди або оголошувати + як змінну або одразу юзати. Хоча тут я б оперетори переніс в константу так як ця змінна реюзається

Reply:
Якщо чесно, думав завернеш, за мою "перевірку на null",

<!--StartFragment-->Double.valueOf(a);        Double.valueOf(b);        Character.valueOf(operation);<!--EndFragment-->

  я іноді навмисно роблю деякі відхилення, скажімо так, щоб зрозуміти, що можна, а що ні.